recipes/open62541/all/conanfile.py

```diff
@@ -12,7 +12,7 @@
 
 class Open62541Conan(ConanFile):
     name = "open62541"
-    license = "MPLv2"
+    license = ("MPL-2.0", "CC0-1.0")
```

Comment:
License IDs need to be listed in https://spdx.org/licenses/. Added missing CC0-1.0 which plugins seems to fall under and that the library seems to package